docs/MigrationGuide.md

```diff
@@ -20,6 +20,8 @@ To see the biggest differences please consult the [changelog](Changelog.md).
   * BestEffortJsonSchemaEncoder moved to package `pl.touk.nussknacker.engine.json.encode` in `nussknacker-json-utils`
 * [#3668](https://github.com/TouK/nussknacker/pull/3668) Method `runWithRequests` of `RequestResponseTestScenarioRunner` (returned by `TestScenarioRunner.requestResponseBased()`)
   now returns `ValidatedNel` with scenario compilation errors instead of throwing exception in that case 
+* [#3701](https://github.com/TouK/nussknacker/pull/3701) Change: SPeL TypedMap Get Accessor:
+  * Right now access to not exist field on TypedMap won't throw exception, just will return `null`
```

Comment:
maybe let's move this description to Changelog, and let's leave migrationGuide as it is now, as in `1.6.1` it works exactly the same.
and also `not exist` -> `not existing`